### ISSUE TITLE
Forward problem in model.py

### ISSUE THREAD

**Issue by bibiaaaa** [tabisheva/keyword-spotting#1]
'conv' is assigned to but never used

        conv = self.cnn(x).permute(0, 2, 1)
        rnn_output, _ = self.rnn(x)
        linear_attn = self.linear(self.attention(rnn_output))
        return torch.log_softmax(linear_attn, dim=1)

**Comment by tabisheva** [comment]
You're right, thank you very much!
